webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+module.exports = {
+    entry: {
+        "boa-sdk": "./src/index.ts",
+        "boa-sdk.min": "./src/index.ts",
+    },
+    devtool: 'source-map',
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
```

Comment:
```suggestion
                test: /\.test.ts?$/,
```
The loader specified in the loader compiles the files that meet the test regular expression conditions.

Reply:
`test: /\.ts?$/` 
I think this is right.